### ISSUE TITLE
Incorrect eslint error for location.hostname with Opera 69

### ISSUE THREAD

**Issue by wulfy** [amilajack/eslint-plugin-compat#406]
hello,

the current line code 
``` export const currentRootDomain = getRootDomain(window.location.hostname); ```

failed during tests with this elsint error : 

```37:48  error  location.hostname() is not supported in Opera 69  compat/compat ``` 

This error is surprising because this property exists for a long time.
I didn't manage to find a documentation about hostname property on Opera 69, but I downloaded Opera 69 (from here http://get.opera.com/ftp/pub/opera/desktop/ ) and tested the result : 

<img width="893" alt="Capture d’écran 2020-09-25 à 20 15 40" src="https://user-images.githubusercontent.com/724239/94302079-ef530380-ff6b-11ea-8f45-5ef6639b44b0.png">

As you can see, Opera 69 supports hostname property

**Comment by wulfy** [comment]
already related in #376

**Comment by Krinkle** [comment]
> already related in #376

Issue #376 seems to be about `window.location` vs bare `location`. However the issue as I see it here is more basic that the rule in question itself appears to be bogus. There is no such thing as a `location.hostname()` method in any web spec. What does exist is the string field `location.hostname` which has existed for a decade including in old and new Opera versions, it certainly  isn't unsupported in Opera 69 or Opera 71, or any other Opera version around that.
